### DISCUSSION TITLE
Performing HPA on a deployment based upon messages in different Azure service bus topics.

### DISCUSSION THREAD

**Discussion by leanprojectslb** [kedacore/keda#3856] [Q&A / Need Help]
I have a scenario in which I need to do HPA of  deployment based on messages in different  Azure service bus topics.
I understand using KEDA we can do HPA based using single TOPIC.

Is it possible to perform HPA using KEDA based upon multiple TOPICS ?

**Comment by zroubalik** [comment]
You can specify multiple triggers in ScaledObject, each can target different topic. The greatest value from all triggers is selected for the scaling decision.

**Reply by tomkerkhove** [reply]
That is correct, learn more on https://keda.sh/docs/2.8/concepts/scaling-deployments/#triggers
